src/components/Sum.jsx

```diff
@@ -1 +1,15 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is `}
+      {a + b}
```

Comment:
```suggestion
      {`Sum of ${a} and ${b} is ${a + b}`}
```

You can also combine it together